yellowbrick/classifier/prcurve.py

```diff
@@ -377,11 +382,17 @@ def precision_recall_curve(model, X, y, ax=None, train_size=0.8,
 
     X : ndarray or DataFrame of shape n x m
         A feature array of n instances with m features the model is trained on.
-        This array will be split into train and test splits.
+        This array will be split into train and test splits if X_test is not specified.
 
     y : ndarray or Series of length n
         An array or series of target or class values. This vector will be split
-        into train and test splits.
+        into train and test splits  if y_test is not specified.
+
+    X_test : ndarray or DataFrame of shape n x m
+        A feature array of n instances with m features that the model is tested on.
```

Comment:
Would you mind adding some language about how this is optional? E.g. 

```
    X_test : ndarray or DataFrame of shape n x m, default=None
        An optional feature array of n instances with m features that the model
        is tested on if specified, using X as the training data. Otherwise 
        X will be split into train and test splits.
```

Reply:
done